src/App.js

```diff
@@ -0,0 +1,127 @@
+import axios from "axios";
+import Header from "./components/Header/Header";
+import Banner from "./components/Banner/Banner";
+import './style/main.css';
+
+import React from 'react';
+
+class App extends React.Component{
+    constructor() {
+        super();
+        this.state = {
+            ingredients: [],
+            startingPrice: "1.00",
+            prices: [],
+            burger: {},
+            order: [],
+            loading: false,
+        }
+    }
+
+    componentDidMount = async () => {
+        try {
+            this.setState({ loading: true });
+            const { data } = await axios.get(
+                "https://burger-api-xcwp.onrender.com/ingredients"
+            );
+
+            const ingredients = data.map((ingredient) =>  ingredient.name);
+
+            const quantities = data.reduce(
+                (acc, curr) => ({ [curr.name]: 0, ...acc }),
+                {}
+            );
+
+            this.setState({
+                prices: data,
+                ingredients: ingredients,
+                burger: quantities,
+                loading: false,
+            });
+        } catch (error) {
+            console.log(error);
```

Comment:
```suggestion
            console.erroe(error);
```

As for me showing a fallback UI its a better idea